source/common/protobuf/utility.h

```diff
@@ -279,6 +279,18 @@ class MessageUtil {
    */
   static void jsonConvert(const Protobuf::Message& source, Protobuf::Message& dest);
 
+  /**
+   * Extract YAML as string from a google.protobuf.Message.
+   * @param message message of type type.googleapis.com/google.protobuf.Message.
+   * @param block_print whether the returned JSON should be in block style rather than flow style.
```

Comment:
Can you add sensible defaults for these (probably true, false respectively)? Most folks "just want some YAML".